README.md

```diff
@@ -1,2 +1,14 @@
 # CodeCoverage
-C++ CodeCoverage
+The CodeCoverage utility is used for generating code coverage reports for C++ repositories. Currently the following LogRhythm repositories can get code coverage measurements through this utility: 
+* [StopWatch](https://github.com/LogRhythm/StopWatch)
+* [Queuenado](https://github.com/LogRhythm/Queuenado)
+* [FileIO](https://github.com/LogRhythm/FileIO)
+* [DeathKnell](https://github.com/LogRhythm/DeathKnell)
+
+## Usage example
+```
+git clone https://github.com/LogRhythm/FileIO
+cd FileIO
+/usr/local/probe/bin/CodeCoverage.py
```

Comment:
@john-gress : something is not working right  then. 
The `UnitTestRunner` explicitly filters out that failing test when running the code coverage, for exactly the reasons you are calling out. 

Ref: https://github.com/LogRhythm/FileIO/blob/master/scripts/unitTestRunner.sh